src/main/scala/cromwell/engine/WorkflowManager.scala

```diff
@@ -59,7 +59,7 @@ trait ActorWorkflowManager extends WorkflowManager {
 
   override def submitWorkflow(wdl: WdlSource, inputs: WorkflowInputs): Try[ManagedWorkflow] = {
     val managedWorkflow = super.submitWorkflow(wdl, inputs)
-    managedWorkflow foreach {_.workflow ! Start(backend)}
+    managedWorkflow foreach {_.workflow ! Start(actorSystem, backend)}
```

Comment:
You shouldn't need to pass an actor system - use context.system (Ugh, pretty sure that thing I committed to develop did exactly this)